packages/sign_in_with_apple/lib/src/widgets/sign_in_with_apple_button.dart

```diff
@@ -20,79 +23,157 @@ class SignInWithAppleButton extends StatelessWidget {
         assert(iconAlignment != null),
         super(key: key);
 
+  /// The callback which will be called when the button is pressed.
   final VoidCallback onPressed;
 
+  /// The text which should be rendered inside the button.
```

Comment:
```suggestion
  /// The text to display next to the Apple log.
```

via https://api.flutter.dev/flutter/widgets/Text/data.html

I think it's clear that this would be inside the button.

I think we can tone down on the conditionals for the normal cases. These things will all happen.